### PR TITLE
Fixed: Gives all coders, maintainers, badmins and hosts in admins.txt a lusty xenomorph maid reskin for the alien queen.

### PR DESCRIPTION
**Pull request by Militaires**
let the mad wars begin

again, this will most likely be the only private reskin i put out really.

### CODE REVIEW

**Comment by Core0verload** [comment]
This is the meme PR to meme all meme PRs.

**Comment by GunHog** [comment]
First: This is a sprites PR with no screenshots in the description.

Second, I disagree with how this is implemented. Code could use improvement, you could probably set the type of xeno to spawn to a var and use that var in the line instead of the hard-coded queen type. That way, you can avoid cluttering the other portions of the code. You did not change the maid's health, so there is no need for you to copy to her subtype.

Third, I feel this feature should be either available to donors, not admins, or simply allow someone to slap a xeno with the maid outfit to give her the maid sprite set.

File: code/modules/mob/living/carbon/alien/humanoid/queen.dm [hunk 1]
```diff
@@ -23,6 +23,18 @@
 	health = 400
 	icon_state = "alienq"
 
+/mob/living/carbon/alien/humanoid/royal/queen/maid
+	name = "alien queen maid"
+	caste = "qmaid"
+	maxHealth = 400
+	health = 400
+	icon_state = "alienqmaid"
```
Comment (Xhuis) [inline code/modules/mob/living/carbon/alien/humanoid/queen.dm]: Shouldn't be a reason to have any differences with the subtype other than the icon state - that, or I don't know xeno code well enough

Comment (Core0verload) [inline code/modules/mob/living/carbon/alien/humanoid/queen.dm]: This is to make xeno maid understand humans.

Comment (GunHog) [inline code/modules/mob/living/carbon/alien/humanoid/queen.dm]: The health does not need to be duplicated.

**Comment by Niknakflak** [comment]
Literally retarded pull request.  Anyone with admin could change their sprite if they wanted to but admins making themselves snowflake is shitty.  The live server doesn't pull from that admin.txt everyone who isn't a real admin on the server won't even be effected unless they boot up a local server in which they could just change their sprite there anyway.  Retarded PR, redundant feature, stupid code.

**Comment by Razharas** [comment]
This needs a proper system for special appearances for special people so we can sell hats like valve

**Comment by Xhuis** [comment]
@Razharas Just give exclusive cosmetics to Patreon donors, the money will just roll in

**Comment by Aloraydrel** [comment]
Pay 20 per month for round start combat gloves and jaunt

**Comment by lzimann** [comment]
no for snowflakes like this

**Comment by pudl** [comment]
ok dis cancer

**Comment by Core0verload** [comment]
If this is getting merged, I'm ready to pay for it with one of my own merge tokens.

**Comment by Core0verload** [comment]
The meme never dies, just evolves.

**Pull request closed without merging**